packages/cojson-storage-sqlite/src/sqliteClient.ts

```diff
@@ -165,12 +166,12 @@ export class SQLiteClient implements DBClientInterface {
   }: { sessionRowID: number; idx: number; signature: Signature }) {
     this.db
       .prepare<[number, number, string]>(
-        `INSERT INTO signatureAfter (ses, idx, signature) VALUES (?, ?, ?)`,
+        "INSERT INTO signatureAfter (ses, idx, signature) VALUES (?, ?, ?)",
       )
       .run(sessionRowID, idx, signature);
   }
 
-  transaction(operationsCallback: () => unknown) {
-    this.db.transaction(operationsCallback)();
+  async transaction(operationsCallback: () => Promise<unknown>) {
+    await this.db.transaction(operationsCallback)();
   }
```

Comment:
This would cause a regression, both functions must be syncronous.

Why are you doing the change?

Reply:
Type warning said async. Let me check

Reply:
Fixed